third_party/ruby/ruby-versions.txt

```diff
@@ -1,2 +1,2 @@
-sorbet_ruby
+sorbet_ruby_2_6
```

Comment:
This will technically affect what artifacts we build and publish in Stripe's CI, right? I think this means we have to be sure that we don't mention `@sorbet_ruby` anywhere in Stripe's codebase anymore (which I think is correct, but I figured I'd call it out).

Reply:
When we upgraded to ruby-2.7 we stripped out all references to the original sorbet_ruby target. I'll grep around and make sure though.